vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -14068,6 +14085,7 @@ unit:MicroMOL-PER-MicroMOL-DAY
   qudt:conversionMultiplier 0.0000115740740740741 ;
   qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T-1D0 ;
```

Comment:
This is an interesting case. The way you did it is fine with me. An alternative is to add a qkdvNumerator of A1 and a qkdvDenominator of A1...T-1. I'm not sure if it adds any value though.

We don't have any rules about how to split up mixed ratios like this.